nuitka/plugins/standard/NumpyPlugin.py

```diff
@@ -219,7 +219,7 @@ def considerExtraDlls(self, dist_dir, module):
             info("")
             info(" Copying extra binaries from 'numpy' installation:")
             for f in binaries:
-                bin_file = f[0].lower()  # full binary file name
+                bin_file = os.path.dirname(f[0]) + os.path.sep + os.path.basename(f[0]).lower()  # full binary file name
```

Comment:
I think using `os.path.sep` and `+` seems not a good idea, but `os.path.join` is always preferable.

But I am wondering, why `lower` is used at all. Is this about filenames in the target to look nicer? I would say that `os.path.normcase` is a much better choice then.